src/lib.rs

```diff
@@ -65,15 +66,18 @@ impl Toasts {
 
     /// Adds new toast to the collection.
     /// By default adds toast at the end of the list, can be changed with `self.reverse`.
+    /// # Panics
+    ///
+    /// Will panic if after adding a toast the list is empty.
+    #[must_use]
     pub fn add(&mut self, toast: Toast) -> &mut Toast {
         if self.reverse {
             self.toasts.insert(0, toast);
             return self.toasts.get_mut(0).unwrap();
-        } else {
-            self.toasts.push(toast);
-            let l = self.toasts.len() - 1;
-            return self.toasts.get_mut(l).unwrap();
         }
+        self.toasts.push(toast);
+        let l = self.toasts.len() - 1;
+        return self.toasts.get_mut(l).unwrap();
```

Comment:
I actually don't know why there is no `clippy::needless_return` warn here as it should be warn by default but make this return implicit please.